backend/Dockerfile

```diff
@@ -4,6 +4,10 @@ WORKDIR /app
 
 EXPOSE 8000
 
+# Install Python dependencies
```

Comment:
NO need to do this again. We install all the requirements in the base image. See, https://github.com/SkywardAI/containers/blob/main/Dockerfile

It contains two part,
* Installing ML libs
* Installing Backend libs.

And this has one most advantage, you do not need to install anything in while you are rebuilding the image in the development env.

The disadvantage, we need to update the base image repo, but it is totally fine for us.